release-notes.md

```diff
@@ -6,6 +6,14 @@ id: release-notes
 
 # {{ page.title }}
 
+## 2.3.0 *November 10, 2015*
```

Comment:
I will fix it also updating the size of the new riot release.. let me merge it once I am done